tests/test_api_views.py

```diff
@@ -60,4 +61,42 @@ def test_institutions(self):
         )
         response = view(request)
         self.assertEqual(response.status_code, 200)
+    def test_exclude_non_normalized_documents(self):
+        view = DocumentList.as_view()
+        create_document(source="bad",normalized=None)
+        create_document(source="good",normalized="This is Normalized")
+        request = self.factory.get(
+            '/documents/'
+        )
+        response = view(request)        
+        self.assertNotContains(response, "bad",
+                            status_code=200)
+        self.assertContains(response, "good", status_code=200)
+        print(response)
+
+
+class APIViewTests2(APITestCase):
```

Comment:
Once you incorporate your code into the document List view, you shouldbe able to add these tests back into the first class - no need for a second